website/docs/examples/camera_aruco_detector/README.md

```diff
@@ -250,7 +250,9 @@ async for event, message in camera_client.subscribe(config.subscriptions[0], dec
 ### 5. Execute the Python script
 
 To run this script from your PC, you need to update the `service_config.json`
-by modifying the `host` filed with your Amiga brain name (e.g., `element-vegetable`).
+by modifying the `host` field with your Amiga brain name.
+Please check out the [Amiga Development 101](../../concepts/system_overview/README.md#where-to-run-the-examples)
```

Comment:
Use doc paths for links. Please fix this here and in all other examples
```suggestion
Please check out the [Amiga Development 101](/docs/concepts/system_overview/#where-to-run-the-examples)
```

Reply:
Done here: b5cef04c9604e21e0de401d3c546fbe0382efd1f